client/lib/purchases/index.ts

```diff
@@ -766,57 +780,3 @@ function shouldRenderMonthlyRenewalOption( purchase ) {
 
 	return false;
 }
-
-export {
```

Comment:
I just exported each of these functions where they are defined, which makes it easier to tell which are public and which are private. The main difference is that if something tried to import the file using CJS syntax (`require()`) it might not work the same way, but I'm pretty sure nothing uses that syntax any more.